framework/src/org/checkerframework/common/value/ReflectiveEvalutator.java

```diff
@@ -94,6 +119,40 @@ public ReflectiveEvalutator(
     }
 
     /**
+     * Convert the arguments that are part of the var arg into an array.
+     *
+     * @param arguments list of arguments
+     * @param method vararg method
+     * @return arguments with varargs converted to an array
+     */
+    private List<Object[]> handleVarArgs(List<Object[]> arguments, Method method) {
+        List<Object[]> newList = new ArrayList<>();
+        int numberOfParameters = method.getParameterTypes().length;
+        for (Object[] args : arguments) {
+            if (args == null) {
+                // null means no arguments.  For varargs no arguments is an empty array.
+                args = new Object[] {};
+            }
+            Object[] newArgs;
+            int numOfVarArgs = args.length - numberOfParameters + 1;
+            if (numOfVarArgs > 0) {
+                newArgs = new Object[numberOfParameters];
```

Comment:
The first two and last 1 line of the then and else bodies are the same.  I would move them outside the `if` statement in order to reduce code duplication.